content/03.combining-datasets.md

```diff
@@ -1,23 +1,28 @@
 
-If a small sample size compromises an ML model's performance, then two approaches can be taken to improve the outcome: 1) increase the number of samples to reduce sparsity, variance, and collinearity, 2) improve the quality of samples to account for sparsity, variance, and collinearity.
-In the first approach, appropriate training, evaluation, and held-out validation sets could be constructed by combining multiple small individual rare disease cohorts [TODO: Link to experimental design box #185].
-In fact, this is often required for the study of rare diseases in the authors' experience.
-In doing so, special attention should be directed towards harmonization since data collection can differ from cohort to cohort.
-Without careful selection of aggregation methods, one may introduce technical variability into the aggregated dataset which can negatively impact the ML model's ability to learn or detect meaningful signal.
-Steps such as reprocessing the data using a single pipeline, using batch correction methods [@doi:10.1093/biostatistics/kxj037; @doi:10.1093/nar/gku864], and normalizing raw values [@doi:10.1186/gb-2010-11-3-r25] may be necessary to mitigate unwanted technical variability.
+When small sample sizes compromise an ML model's performance, then two approaches can be taken to improve the outcome: 1) increase the number of samples to reduce sparsity, variance, and collinearity, 2) improve the quality of samples to account for sparsity, variance, and collinearity.
+In the first approach, appropriate training, evaluation, and held-out validation sets could be constructed by combining multiple small individual rare disease cohorts (Fig.1b, Box 1).
+In the authors' experience, this is often a neccessity in case of rare diseases.
+In combining datasets, special attention should be directed towards _data harmonization_ since data collection methods can differ from cohort to cohort.
+Without careful selection of aggregation methods, one may introduce technical variability into the combined dataset which can negatively impact the ML model's ability to learn or detect meaningful signal.
+Steps such as reprocessing the data using a single pipeline, using batch correction methods [@doi:10.1093/biostatistics/kxj037; @doi:10.1093/nar/gku864], and normalizing raw values [@doi:10.1186/gb-2010-11-3-r25] may be necessary to mitigate unwanted technical variability. (Figure {@fig:1}a)
 
-In the second approach, small but meaningfully generated datasets can greatly enhance the performance of ML models in the context of rare disease.
+In the second approach, small but meaningfully generated datasets can greatly enhance the performance of ML models too.
 Specifically, improving labeling of data is critical in accounting for sparsity and variance in the data. 
 In our experience, collaboration with domain experts has proved to be critical in gaining insight into potential sources of variation in the datasets.
```

Comment:
Otherwise we use "critical" two sentences in a row

```suggestion
In our experience, collaboration with domain experts has proved to be essential in gaining insight into potential sources of variation in the datasets.
```